src/header/header.scss

```diff
@@ -274,61 +227,72 @@
 
   .iui-header-button-icon {
     padding: $iui-xs;
+    box-sizing: content-box;
     @include iui-header-button-icon($iui-icons-default, $iui-border-radius);
     @include themed {
       background-color: t(iui-color-background-3);
       fill: t(iui-icons-color);
     }
   }
 
+  img.iui-header-button-icon {
```

Comment:
do we have example with img as button icon?

Reply:
We do not currently but I could add one.  I did just now test it locally and found 2 issues, so we probably should add a test for this.

Reply:
Also worth pointing out that our react header is using `<img>` & these changes will fix the display bug in react:

**React**
![Screen Shot 2022-02-18 at 4 17 13 PM](https://user-images.githubusercontent.com/849817/154762323-75c58256-cb7a-49f4-b855-bcea91ac9c90.png)

**CSS**
![Screen Shot 2022-02-18 at 4 17 01 PM](https://user-images.githubusercontent.com/849817/154762346-91ebf77a-3646-4c1a-a690-7b49b2d2fa6a.png)